wordpress-mobile/WordPress-Android
# Fix Navigation Bar Buttons Not Visible with Signup Sheet

This PR tries to fix  #10908

It is an existing [Android issue](https://issuetracker.google.com/issues/65477640) which has become obsolete because of lower priority.  Seems like we need to go for a hack to fix it. Tried different [solutions](https://stackoverflow.com/questions/47553936/prevent-bottomsheetdialogfragment-covering-navigation-bar), this one seems to be working well in both portrait and landscape mode.

### To test:
1. Clear app data.
2. Launch app.
3. Tap Sign Up for WordPress.com button.
4. Notice navigation buttons are visible.

| API | Orientation | Before | After |
| ------ | ------------- | ------------- | ------------- |
| 27 | Portrait |  ![API27Before](https://user-images.githubusercontent.com/1405144/71882354-56205a80-315a-11ea-9b78-414405a129b4.png)|![API27After](https://user-images.githubusercontent.com/1405144/71882496-94b61500-315a-11ea-894b-5e9860a2dacc.png)|
| 27 | Landscape | ![API27BeforeLandscape](https://user-images.githubusercontent.com/1405144/71882742-0a21e580-315b-11ea-853b-a40785501f10.png)| ![API27AfterLandscape](https://user-images.githubusercontent.com/1405144/71882772-17d76b00-315b-11ea-922e-8503583bbb19.png)|
| 29 | Portrait | ![Android10BeforePortrait](https://user-images.githubusercontent.com/1405144/71882898-4a816380-315b-11ea-8c9b-d0774baf246b.png)|![Android10AfterPortrait](https://user-images.githubusercontent.com/1405144/71883051-992efd80-315b-11ea-9bfe-14b10b3ad8b8.png) |
| 29 | Landscape | ![Android10BeforeLandscape](https://user-images.githubusercontent.com/1405144/71883501-954fab00-315c-11ea-8132-b40b8ddb18be.png) | ![Android10AfterLandscape](https://user-images.githubusercontent.com/1405144/71883527-9e407c80-315c-11ea-8fcc-36aeb382ad05.png)|



PR submission checklist:
